<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogfsCreate</class>
 <widget class="QDialog" name="DialogfsCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>377</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>395</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Filesystem</string>
  </property>
  <property name="windowIcon">
   <iconset resource="zmanagerres.qrc">
    <normaloff>:/icons/kexi.png</normaloff>:/icons/kexi.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="15" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="fsName"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QFrame" name="frameProps">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Properties override at creation time:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="spinCopies">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkCanMount">
        <property name="text">
         <string>Prevent auto mount (canmount=noauto)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkMountpt">
        <property name="text">
         <string>Mountpoint</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="mountPtEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboMountpt">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>none</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>legacy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>[path]</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkUtf8Only">
        <property name="text">
         <string>Force UTF-8 only (utf8only=on)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkNormalization">
        <property name="text">
         <string>Unicode normalization =</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboNormalization">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>none</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>formD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>formKCF</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="checkCopies">
        <property name="text">
         <string>Copies =</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="checkDedup">
        <property name="text">
         <string>Deduplication (dedup=on)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="checkCompression">
        <property name="text">
         <string>Compression =</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="comboCompression">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>on</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>lzjb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>zle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gzip</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gzip-1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gzip-2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gzip-3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gzip-4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gzip-5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gzip-6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gzip-7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gzip-8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gzip-9</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="zmanagerres.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogfsCreate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogfsCreate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
